### PR TITLE
[12.0][ADD] hr_timesheet_analysis: analyze tracked time in Pivot, Graph views

### PR DESCRIPTION
**Pull request by alexey-pelykh**
image
image

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@nikul-serpentcs @aheficent @bjeficent it would be great to hear your thoughts on this, if you don't mind of course

File: hr_timesheet_analysis/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
Comment (nikul-serpentcs) [inline hr_timesheet_analysis/tests/__init__.py]: @alexey-pelykh If not any tests then remove a file.

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor change

File: hr_timesheet_analysis/tests/__init__.py (hunk 1 above)
Comment (alexey-pelykh) [reply hr_timesheet_analysis/tests/__init__.py]: done

**Comment by alexey-pelykh** [comment]
@nikul-serpentcs changes made, please re-review 👍

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Review by bjeficent — APPROVED** [review]
Functional test :+1:

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional tests ok. Just a question. Is it possible to create a new menu entry under "Reporting" menu?

**Comment by alexey-pelykh** [comment]
@aheficent sure, yet what's the value of it? View buttons are located at default place, thus no explicit menu item is needed. Actually I don't even know why Odoo disabled pivot button yet added 4 various items in reporting - it's not a reporting per-se

**Comment by AaronHForgeFlow** [comment]
I commented that just because we have under reporting menu some graph views and this could be another one, just to keep the same format. But it is not blocking just to see what people think here.

**Comment by alexey-pelykh** [comment]
@aheficent I think I'll extend this module with some proper true reports in next quarter, those surely would be in Reporting menu

**Pull request merged**